package com.interviewlab.exception;

public class InsufficientBalanceException extends PaymentException {

    public InsufficientBalanceException(String message) {
        super(message);
    }

    public InsufficientBalanceException(String message, Throwable cause) {
        super(message, cause);
    }
}
